test_project/test_project/settings.py

```diff
@@ -0,0 +1,102 @@
+"""
+Django settings for test_project project.
+
+Generated by 'django-admin startproject' using Django 1.8.8.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+"""
```

Comment:
There's quite a lot of cruft in this settings file - have you seen [the one in `base-library`](https://github.com/incuna/base-library/blob/master/test_project/settings.py)?